src/solver/optimisation/ConsistenceNODU.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include "constraint_builder.h"
+
+class ConsistenceNODU : private Constraint
```

Comment:
I don't know if everyone would agree, but I would prefer the verbose name instead of `NODU`

Reply:
@flomnes @guilpier-code

Reply:
I agree with @sylvlecl !